backend/app/core/exceptions/handlers.py

```
@@ -1,42 +1,102 @@
 import logging
-
-from fastapi import Request, HTTPException
+import traceback
+from fastapi import Request
 from fastapi.exceptions import RequestValidationError
 from fastapi.responses import JSONResponse
 
 from app.core.exceptions.types import AppError
 from app.core.response.response import error_response
```

Flake8 (7.2.0)

[error] 'app.core.response.response.error_response' imported but unused

(F401)